lib/osf-components/addon/components/file-icon/template.hbs

```diff
@@ -1 +1,6 @@
-{{fa-icon this.iconName fixedWidth=true}}
+<FaIcon
+    data-test-file-icon
+    @icon={{this.iconName}}
+    @prefix={{if (eq this.iconName 'cube') 'fas' 'far'}}
```

Comment:
`cube` is not available in the default set of icons, so I had to have it grab from another set.